packages/extension/src/ui/services/voyager.service.ts

```diff
@@ -27,3 +27,11 @@ export const getVoyagerTransactionLink = (
 export const openVoyagerTransaction = (hash: string, network: Network) => {
   window.open(getVoyagerTransactionLink(hash, network), "_blank")?.focus()
 }
+
+export const openVoyager = (network: Network, address?: string) => {
```

Comment:
this is not correct. if there is no address this function shouldn't be called
```suggestion
export const openVoyagerAddress = (network: Network, address: string) => {
```

Reply:
this remains